Add Project Loved: Week of June 9th news post (#2303)
ppy/osu-wiki: news/2019-06-12-project-loved-week-of-june-9th.md

296 BPM, 5:46 | 5.60★  
*description written by [fartownik](https://osu.ppy.sh/users/56917)*

A classic from 2013 comes back to life 6 years later in this Project Loved round! Jiyuu e no Sanka is a single-difficulty collab between two old-school mappers, RLC and Nyquill. The map is heavily slider-based, with the circle-slider ratio being at around 1:2. It doesn't make it any easier though—quite the opposite! The usage of slider tick rate 2 and tricky repeating sliders make it so almost every slider has to be perfectly followed in order not to break your combo. Take a break from your standard pp map and have a go at something a bit unusual!

[![](/wiki/shared/news/2019-06-12-project-loved-week-of-june-9th/osu/7-denden-passion.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)

[Dempagumi.inc - Denden Passion](https://osu.ppy.sh/beatmapsets/419144#osu) by [Yoari](https://osu.ppy.sh/users/4160699), [Justaway12](https://osu.ppy.sh/users/4445283) and [ErunamoJAZZ](https://osu.ppy.sh/users/1869764)  
157 BPM, 3:51 | 6.05★, 6.51★, 6.53★, 7.13★  
*description written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*

> If you're looking for a classic, you've come to the right place. Den Den passion is a map that was enjoyed by many through only word of mouth. It's a mapset that's just really fun to play in general with its cheerful and upbeat rhythm. Its top difficulty gives players a test for their aim with its large spacing jumps and dense streams. Included in this mapset are a couple guest difficulties: one from Justaway12 which focuses more on symmetrical jumps, and another from ErunamoJAZZ which is more of a map for players to just enjoy their time. All in all, it's hard to leave this map without a smile :).

Reviewer: ```suggestion
If you're looking for a classic, you've come to the right place. Denden passion is a map that was enjoyed by many through only word of mouth. It's a mapset that's just really fun to play in general with its cheerful and upbeat rhythm. Its top difficulty gives players a test for their aim with its large spacing jumps and dense streams. Included in this mapset are a couple guest difficulties: one from Justaway12 which focuses more on symmetrical jumps, and another from ErunamoJAZZ which is more of a map for players to just enjoy their time. All in all, it's hard to leave this map without a smile :).
```

[D(ABE3) - MANIERA](https://osu.ppy.sh/beatmapsets/336828#osu) by [SakiZ](https://osu.ppy.sh/users/4347191) et al.  
208 BPM, 2:00 | 2.55★ – 8.00★  
*description written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*

D(ABE3)'s "MANIERA" was quick on its way to popularity upon getting its first ranked [osu!mania mapset](https://osu.ppy.sh/beatmapsets/272871) back in 2015. Shortly after, an osu!standard mapset was uploaded, giving players an opportunity to take a swing at this beast of a song. Thanks to Cookiezi's FC on [ATHE'S MASTERPIECE] (now ATHE'S ULTRA), the map received a lot of attention, and now it's here in Project Loved. With guest difficulties from AtHeoN, Mark101, Hikan, Hokichi, SkyDevil and ReFaller on top of SakiZ's [THREE DIMENSIONS], there is surely a difficulty for a player of any skill.

[![](/wiki/shared/news/2019-06-12-project-loved-week-of-june-9th/osu/9-future-cider.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)

[YUC'e - Future Cider](https://osu.ppy.sh/beatmapsets/607233#osu) by [Kaifin](https://osu.ppy.sh/users/2596942)  
180 BPM, 2:12 | 7.45★  
*description written by [-Roxas](https://osu.ppy.sh/users/1986262)*

> With Kaifin's new album "Duality" having dropped on the 7th of June ([be sure to check it out!](https://twitter.com/kaifinowo/status/1136845697141993473)), what's a better time to have a Kaifin map appear in Loved? If you have played any of Kaifin's maps, you'll know that they're far from standard and get very very technical. "Future cider" really pushes your reading skill with many complicated patterns, including spaced bursts and triples, fast and slow sliders, and a few slider streams chucked in here and there. This map isn't 7.4 stars for nothing, and it'll truly test your abilities as a player. Good luck if you decide to attempt this map!

Reviewer: ```suggestion
With Kaifin's new album "Duality" having dropped on the 7th of June ([be sure to check it out!](https://twitter.com/kaifinowo/status/1136845697141993473)), what's a better time to have a Kaifin map appear in Loved? If you have played any of Kaifin's maps, you'll know that they're far from standard and get very very technical. "Future Cider" really pushes your reading skill with many complicated patterns, including spaced bursts and triples, fast and slow sliders, and a few slider streams chucked in here and there. This map isn't 7.4 stars for nothing, and it'll truly test your abilities as a player. Good luck if you decide to attempt this map!
```

日本が誇るSV mapperのiceOCが手掛ける高難易度譜面が今週のpickに登場します！この譜面は、SVがとても多彩ながらも、初見でも繋ぐことが出来るような予測性があります。これは初見の人が多いマルチで投げても怒られないほどです。多分。ただ、プレイヤーのFCを阻むのは、この譜面の高密度な連続した連打です。SVと高密度なパターンを乗り越え、マッパーからフルコンボを勝ち取ることを願っています！

## <a name="osucatch" id="osucatch"></a>osu!catch

osu!catch Loved candidates were chosen by [Sartan](https://osu.ppy.sh/users/4100941), [Tenshichan](https://osu.ppy.sh/users/1101600), [Wesley](https://osu.ppy.sh/users/2407265) and [Yumeno Himiko](https://osu.ppy.sh/users/1806962)!

---

[![](/wiki/shared/news/2019-06-12-project-loved-week-of-june-9th/catch/1-augoeides.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)

[DystopiaGround - AugoEidEs](https://osu.ppy.sh/beatmapsets/191008#fruits) by [Serena-](https://osu.ppy.sh/users/2193444), [Minato Yukina](https://osu.ppy.sh/users/531253), [ExGon](https://osu.ppy.sh/users/214187), [Frobe](https://osu.ppy.sh/users/670365), [Last Bubble](https://osu.ppy.sh/users/1302259) and [Spectator](https://osu.ppy.sh/users/702598)  
207 BPM, 6:51 | 6.08★, 6.14★  
*description written by [Wesley](https://osu.ppy.sh/users/2407265)*

> AugoEidEs is a song most osu!catch players will know very well, but this time the map is created by several Korean players. The mapset has two fully-mapped difficulties. "Crystal" is made by Minato Yukina, while "Darkness" is a collaboration by Serena-, ExGon, Frobe, Minato Yukina and Last Bubble where Spectator handled the hitsounds. Both difficulties are what you would expect of such an intense song: a lot of streams, jumps and of course a lot of creativity. As the map is somewhat old already (2014), it's a lot different compared to the newer AugoEidEs mapped by Koliron. Expect to see a lot of "older" stream patterns and jumps while playing either of these two difficulties!

Reviewer 2: ```suggestion
AugoEidEs is a song most osu!catch players will know very well, but this time the mapset's two difficulties are created by several Korean players. The difficulty "Crystal" is made by Minato Yukina, while "Darkness" is a collaboration by Serena-, ExGon, Frobe, Minato Yukina and Last Bubble where Spectator handled the hitsounds. Both difficulties are what you would expect of such an intense song: many streams, jumps and of course a lot of creativity. As the map is somewhat old already (2014), it's a lot different compared to the newer AugoEidEs mapped by Koliron. Expect to see a lot of "older" stream patterns and jumps while playing either of these two difficulties!
```
removed redundancy and clarified what "Crystal"/"Darkness" is

[![](/wiki/shared/news/2019-06-12-project-loved-week-of-june-9th/catch/2-all-i-need-are-things-i-like.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)

[nameless - All I Need are Things I Like](https://osu.ppy.sh/beatmapsets/742694#fruits) by [Alostis](https://osu.ppy.sh/users/2986875)  
140 BPM, 4:17 | 7.17★  
*description written by [Sartan](https://osu.ppy.sh/users/4100941)*

> Another round, another first timer for the osu!catch Loved vote! Here we have Alostis with his most popular map, All I Need are Things I Like. Despite the very small quantity of maps he has created, Alostis has put out a true banger with this second map of his. For its time of release last year, this map has been able to accumulate quite a respectable amount of favourites from players, and with good reason. The combination of the cute and funky song, together with very enjoyable jumping patterns for players on a pretty high level has made this map quite a success, with it appearing in multiplayer lobbies quite often. Mods can also very well be applied to this map, with no patterns becoming clunky with HardRock and Hidden being quite friendly thanks to the pretty high AR of 9.6. So go check it out, if you need new things to like!

Reviewer 2: ```suggestion
Another round, another first timer for the osu!catch Loved vote! Here we have Alostis with his most popular map, All I Need are Things I Like. Despite the very small quantity of maps he has created, Alostis has put out a true banger with this second map of his. Since its time of release last year, this map has been able to accumulate quite a respectable amount of favourites from players, and for good reason. The combination of the cute and funky song, together with very enjoyable jumping patterns for players on a pretty high level has made this map quite a success, with it appearing in multiplayer lobbies quite often. Mods can also very well be applied to this map, with no patterns becoming clunky with HardRock and Hidden being quite friendly thanks to the pretty high AR of 9.6. So go check it out, if you need new things to like!
```
grammar

The Extra difficulty still holds up quite well today as a sporadic yet straightforward experience, and there have been many players who have enjoyed this chart over the past few years noting how gratifying it is to play through the chart and its song choice. If you're into a lightly technical chart with strong progression, this chart is most definitely up your alley.

[![](/wiki/shared/news/2019-06-12-project-loved-week-of-june-9th/mania/2-one-day-sky.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)

[Suzuki Konomi - One Day Sky](https://osu.ppy.sh/beatmapsets/621883#mania) by [shuniki](https://osu.ppy.sh/users/4942174)  
118 BPM, 4:39 | [4K] 6.94★, [4K] 7.78★

> Shuniki's back once again with anything and everything we've come to know and love about him. Intense O2Jam-esque long note mapping, insanely difficult and varied patterns, and an immaculate popularity matched by nearly no one in the long note mapping community. This time being brought to Loved for his map of One Day Sky, a representation of all that was mentioned previously, but with a massive boost in the difficulty compartment when compared to pretty much every other ln chart nominated to ranked or loved. The set's highest difficulty is nothing to scoff at difficulty wise, utilizing extremely dense and awkward minijack and ln patterns in the chorus, and giving even the best of the best a challenge to face. The lower difficulty, however, is a much easier and approachable (albeit still very daunting) experience, boasting many of the same lovable qualities, just with easier, more comfortable patterning throughout. This is a pick representative of the truly hardcore LN fans, and certainly one not to miss by anyone looking for more LN in their lives.

Reviewer 2: ```suggestion
Shuniki's back once again with anything and everything we've come to know and love about him. Intense O2Jam-esque long note mapping, insanely difficult and varied patterns, and an immaculate popularity matched by nearly no one in the long note mapping community. This time being brought to Loved for his map of One Day Sky, a representation of all that was mentioned previously, but with a massive boost in the difficulty department when compared to pretty much every other LN chart nominated to Ranked or Loved. The set's highest difficulty is nothing to scoff at difficulty wise, utilizing extremely dense and awkward minijack and LN patterns in the chorus, and giving even the best of the best a challenge to face. The lower difficulty, however, is a much easier and approachable (albeit still very daunting) experience, boasting many of the same lovable qualities, just with easier, more comfortable patterning throughout. This is a pick representative of the truly hardcore LN fans, and certainly one not to miss by anyone looking for more LN in their lives.
```
grammar and osuword standardization

[![](/wiki/shared/news/2019-06-12-project-loved-week-of-june-9th/mania/3-didj-pvc.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)

[The Flashbulb - DIDJ PVC](https://osu.ppy.sh/beatmapsets/884240#mania) by [Vortex-](https://osu.ppy.sh/users/4999669)  
135 BPM, 1:26 | [4K] 3.69★

> From one of osu!mania's more recognised modern charters today, DIDJ PVC is a slightly more technical product from what you'd expect from Vortex- given his previous charts in the Loved section. The chart is very technical despite its low density, with many minijacks, gallops, substantial LN use, and very few continuous streams. The chart itself goes to what practically sounds like a drum solo, and the chart almost feels like playing on a snare drum (given its heavy use of minijacks and two-hand trills), which emulates the percussion of the song nicely.

Reviewer 2: ```suggestion
From one of osu!mania's more recognised modern charters today, DIDJ PVC is a slightly more technical product from what you'd expect from Vortex- given his previous charts in the Loved section. The chart is very demanding despite its low density, with many minijacks, gallops, substantial LN use, and very few continuous streams. The chart itself goes to what practically sounds like a drum solo, and the chart almost feels like playing on a snare drum (given its heavy use of minijacks and two-hand trills), which emulates the percussion of the song nicely.
```
redundancy

> From a structural standpoint, this chart has the usual chord allocation and column allocation techniques that you'd see from Vortex (pay attention to whether inner or outer column singles go toward certain sounds), LNs with lengths that depend on the type of sound (higher louder sound = longer LN) with occasional use of pitch relevance and grace notes whenever possible, and an overall tactful use of minijacks. What I've mentioned sounds pretty simple on paper, but thanks to the track's near-unpredictable structure, the chart itself is a crazy dynamic experience with many spikes and turns that will keep many players on their feet (or fingers) throughout the entire chart.

Reviewer 2: Vortex- (needs the -)

[![](/wiki/shared/news/2019-06-12-project-loved-week-of-june-9th/mania/4-tsubomi-long-version.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)

[Sano Hiroaki feat. Tatsumi Megumi - Tsubomi (Long Version)](https://osu.ppy.sh/beatmapsets/349611#mania) by [moni](https://osu.ppy.sh/users/1750531)  
184 BPM, 5:07 | [7K] 8.01★

> A chart bearing the hallmark of O2Jam converts, Tsubomi has it all for anyone looking for a fun and noodly time. Coming in at 197 BPM, players are to expect red line speed changes and insanely difficulty LN usages. Visuals are given a very strong emphasis, giving each chorus its own character with the help of playful manipulation of LN lengths and chord sizes. Beware the final section, as accuracy difficulties are ramped up with the introduction of 1/8 LNs that take the place of rice notes. This is a must-try for noodle slurpers!

Reviewer 2: ```suggestion
A chart bearing the hallmark of O2Jam converts, Tsubomi has it all for anyone looking for a fun and noodly time. Coming in at 197 BPM, players are to expect tempo changes and insanely difficulty LN usages. Visuals are given a very strong emphasis, giving each chorus its own character with the help of playful manipulation of LN lengths and chord sizes. Beware of the final section, as accuracy difficulties are ramped up with the introduction of 1/8 LNs that take the place of rice notes. This is a must-try for noodle slurpers!
```
red line -> tempo change and grammar

[![](/wiki/shared/news/2019-06-12-project-loved-week-of-june-9th/mania/5-niwaka-sannen-ota-hachinen-internets-wa-forever-choubun-sumaso-long-ver.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)

[Camellia feat. Nanahira - Niwaka Sannen Ota Hachinen, Internets wa forever (Choubun Sumaso Long ver.)](https://osu.ppy.sh/beatmapsets/748377#mania) by [Kim_GodSSI](https://osu.ppy.sh/users/2218047)  
216 BPM, 5:04 | [7K] 7.13★

> It's time to throw in another 7K marathon, and we're going in with a blast with a Camellia chart! Primarily a dense and rolly chart, the map primarily puts an emphasis on huge 126 BPM 1/2 chords and 1/4 chordtrills with rolly chordstreams. They're are a staple in this chart which highlights the chaotic nature of the song. While the chart can be a little dense to deal with, a generous break section is followed by a very aggressive delay-like dubstep section to bring back the pacing of the song. The chart is finally topped off with the same general flow but with much more dense chord usages! This is generally a fun pick for anyone wanting to play a marathon.

Reviewer 2: ```suggestion
It's time to throw in another 7K marathon, and this time we're aiming for a blast with a Camellia chart! Primarily a dense and rolly chart, the map mainly puts an emphasis on huge 126 BPM 1/2 chords and 1/4 chordtrills with rolly chordstreams. They're are a staple in this chart which highlights the chaotic nature of the song. While the chart can be a little dense to deal with, a generous break section is followed by a very aggressive delay-like dubstep section to bring back the pacing of the song. The chart is finally topped off with the same general flow but with much more dense chord usages! This is generally a fun pick for anyone wanting to play a marathon.
```
redundancies
